File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/good/RecommendedProducts.java

package com.artmurka.artmurkaapp.data.model.pojo.itemlist.good;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RecommendedProducts {

    @SerializedName("recomm_products")
    @Expose
    private RecommProducts recommProducts;
    @SerializedName("present")
    @Expose
    private Integer present;

    public RecommProducts getRecommProducts() {
        return recommProducts;
    }

    public void setRecommProducts(RecommProducts recommProducts) {
        this.recommProducts = recommProducts;
    }

    public Integer getPresent() {
        return present;
    }

    public void setPresent(Integer present) {
        this.present = present;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/good/RecommendedProducts.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.good

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RecommendedProducts {

    @SerializedName("recomm_products")
    @Expose
    var recommProducts: RecommProducts? = null
    @SerializedName("present")
    @Expose
    var present: Int? = null

}
